spec/controllers/reports_controller_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe ReportsController, type: :controller do
+  let!(:user){ create :user }
+  before :each do
+    sign_in user
+  end
+
+  describe 'GET #index' do
+    let!(:first_report){ FactoryGirl.create(:report, user: user)}
+    let!(:second_report){ FactoryGirl.create(:report, user: user)} 
+    
+    
+    it 'populate an array of all reports @reports' do
+      get :index
+      expect(assigns(:reports)).to match_array([second_report, first_report])
+    end
+
+    it 'have a positive responce status 200' do
+      get :index
```

Comment:
So if you move get :index you can simply write 
it { expect(response.status).to eq(200) }